sim/Makefile

```diff
@@ -12,17 +12,15 @@ DUT = tb_didactic
 
 TB_DEFINES = \
 	+define+USE_UART \
+	+define+INC_ASSERT
```

Comment:
this is not needed, it applies only to ibex if it is compiled without `SYNTHESIS` flag.

Reply:
This raises the question if simulations run with `SYNTHESIS` defined are reasonable. For example, using the Ibex instruction tracer requires `RVFI` to be defined, which I can foresee conflicting with the `SYNTHESIS` flag.

Also, utilizing the asserts that are already provided is good practice, no?